src/EditorFeatures/CSharpTest/Formatting/FormattingEngineTestBase.cs

```diff
@@ -172,10 +170,18 @@ internal static void AssertFormat(Workspace workspace, string expected, OptionSe
             Assert.Equal(expected, actual);
         }
 
-        protected static void AssertFormatWithView(string expectedWithMarker, string codeWithMarker, bool debugMode = false)
+        protected static void AssertFormatWithView(string expectedWithMarker, string codeWithMarker, bool debugMode = false, params (PerLanguageOption<bool>, bool)[] options)
```

Comment:
you could use new tuple feature where you can give name to tuple (Option option, bool enabled)

Reply:
thanks